core-test/.jvm/src/test/scala/org/bitcoins/core/p2p/InetAddressTest.scala

```diff
@@ -0,0 +1,31 @@
+package org.bitcoins.core.p2p
+
+import org.bitcoins.testkitcore.gen.p2p.P2PGenerator
+import org.bitcoins.testkitcore.util.BitcoinSUnitTest
+
+import java.net.{InetAddress => JvmAddress}
+
+class InetAddressTest extends BitcoinSUnitTest {
+
+  implicit override val generatorDrivenConfig: PropertyCheckConfiguration =
+    generatorDrivenConfigNewCode
+
+  behavior of "InetAddressTest"
+
```

Comment:
is there any hard coded tests we can just add that is run across both runtimes? 

Even some simple ones would be good, because with these tests they can't be run on js

Reply:
Added some